python/vpc/vpc-flow-logs/app.py

```diff
@@ -0,0 +1,47 @@
+from aws_cdk import (
+    core,
+    aws_ec2 as ec2,
+    aws_iam as iam,
+    aws_logs as logs
+)
+
+class VpcFlowStack(core.Stack):
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        
+        # Setup IAM user for logs
+        vpc_flow_role = iam.Role(
+            self, 'FlowLog',
+            assumed_by=iam.ServicePrincipal('vpc-flow-logs.amazonaws.com')
+        )
+
+        # Create Cloudwatch log group
+        log_group = logs.LogGroup(
+            self, 'LogGroup',
+            log_group_name='vpc-flow-example',
```

Comment:
Hard-naming resources is a pattern that we don't want to encourage. Please let names be automatically generated.